samples/add-features-with-contingent-values/src/main/java/com/esri/arcgismaps/sample/addfeatureswithcontingentvalues/screens/MainScreen.kt

```diff
@@ -0,0 +1,292 @@
+            }
+        }
+    )
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun BottomSheetContents(
+    mapPoint: Point?,
+    bottomSheetState: SheetState,
+    onBottomSheetStateChange: (SheetState) -> Unit,
+    uiState: UIState,
+    statusAttributes: List<CodedValue>,
+    onStatusAttributeSelect: (CodedValue) -> Unit,
+    protectionAttributes: List<CodedValue>,
+    onProtectionAttributeSelect: (CodedValue) -> Unit,
+    sliderControlParameters: SliderControlParameters,
+    onBufferSizeSelect: (Int) -> Unit,
+    onApplyButtonClicked: (Point) -> Unit,
+) {
+    val coroutineScope = rememberCoroutineScope()
+    Column(
+        modifier = Modifier.padding(start = 12.dp, end = 12.dp, bottom = 12.dp),
+    ) {
+        Text(
+            text = "Add Feature",
+            style = MaterialTheme.typography.titleMedium,
+            modifier = Modifier.align(Alignment.CenterHorizontally)
+        )
+
+        Spacer(Modifier.size(8.dp))
+        Text("Attributes:")
+
+        // dropdown boxes for selecting feature status/protection
+        AttributeDropdown("Status", uiState.status, statusAttributes, onStatusAttributeSelect)
+        AttributeDropdown("Protection", uiState.protection, protectionAttributes, onProtectionAttributeSelect)
+        Spacer(Modifier.size(8.dp))
+
+        // buffer size displayed and updated in slider
+        var localBufferSize by remember { mutableIntStateOf(uiState.buffer) }
+
+        // update buffer size if it changes in the view model
+        var previousBufferSize by remember { mutableIntStateOf(0) }
+        if (uiState.buffer != previousBufferSize) {
+            previousBufferSize = uiState.buffer
+            localBufferSize = uiState.buffer
+        }
+
+        // recenter the slider if contingent values change
+        var bufferRange = sliderControlParameters.minRange..sliderControlParameters.maxRange
+        if (!bufferRange.contains(localBufferSize)) {
+            localBufferSize = (bufferRange.start + bufferRange.endInclusive) / 2
+        }
+
+        Row {
+            Text("Exclusion area buffer size:")
+            Spacer(Modifier.weight(1f))
+            Text(text = if (localBufferSize > 0) localBufferSize.toString() else "")
+        }
+        Slider(
+            enabled = sliderControlParameters.isEnabled,
+            value = localBufferSize.toFloat(),
+            valueRange = sliderControlParameters.minRange.toFloat()..sliderControlParameters.maxRange.toFloat(),
+            steps = (bufferRange.endInclusive - bufferRange.start).toInt(),
+            onValueChange = { localBufferSize = it.roundToInt() },
+            onValueChangeFinished = { onBufferSizeSelect(localBufferSize) },
+            track = { sliderState ->
+                SliderDefaults.Track(
+                    enabled = sliderControlParameters.isEnabled,
+                    sliderState = sliderState,
+                    drawStopIndicator = null,
+                    drawTick = { _, _ -> }
```

Comment:
Can these two arguments be ommitted?
```suggestion
```

Reply:
I've set the `steps` parameter so the slider snaps to integer values rather than being continuous. By default, that adds a marker to the slider for every tick - quite a lot of markers for the range I'm passing! These two parameters remove the markers.